keentools_facebuilder/settings.py

```diff
@@ -506,6 +522,20 @@ def uv_items_callback(self, context):
     return res
 
 
+def model_level_callback(self, context):
```

Comment:
What is this code for? Do not use names as constants anywhere

Reply:
Use fb.model_names

Reply:
model_level_callback generates enum values from fb.model_names
Hardcoded dict is used for additional tooltips and icons if there are known names detected via fb.model_names. In other case there will be fb.mode_names without tooltips and custom icons.

Reply:
Remove Hardcoded dict. If we need additional tooltips we can add them in fb.model_tooltips